docs/en/sql-reference/data-types/aggregatefunction.md

```diff
@@ -1,5 +1,5 @@
 ---
-toc_priority: 52
+toc_priority: 53
```

Comment:
What does this number mean?

Reply:
This is the position of a topic in the left Contents panel.